docs/rest-api/rest-interpreter.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: page
+title: "Interpreter REST API"
+description: ""
+group: rest-api
+---
+{% include JB/setup %}
+
+## Zeppelin REST API
+ Zeppelin provides several REST API's for interaction and remote activation of zeppelin functionality.
+ 
+ All REST API are available starting with the following endpoint ```http://[zeppelin-server]:[zeppelin-port]/api```
+ 
+ Note that zeppein REST API receive or return JSON objects, it it recomended you install some JSON view such as 
+ [JSONView](https://chrome.google.com/webstore/detail/jsonview/chklaanhfefbnpoihckbnefhakgolnmc?utm_source=chrome-app-launcher-info-dialog)
```

Comment:
If we want to keep a link to the external resource, may be we should remove the Google Analytics tracking part at least `?utm_source=chrome-app-launcher-info-dialog`